exercises/I-array-properties/exercise.js

```diff
@@ -6,9 +6,15 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  if(arr.length) {
```

Comment:
would you be able to simplify this so that it uses one line return statement instead of an if/else statement?